platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py

```diff
@@ -68,6 +71,25 @@ def _value_get(d: dict, key_prefix, key_suffix=''):
 
 # Thermal -> ThermalBase -> DeviceBase
 class Thermal(ThermalBase):
+    _thresholds = {
```

Comment:
Will it work for Montara/Maveriks ?

Reply:
I am not sure. For final solution about Montara to need testing and maybe additional records.

Reply:
Please do it